src/Plugin/GraphQL/DataProducer/Entity/EntityLoad.php

```diff
@@ -134,6 +147,14 @@ public function resolve($type, $id = NULL, $language = NULL, $bundles = NULL, Fi
         return NULL;
       }
 
+      // Check if the passed user (or current user if none is passed) has access
+      // to the entity, if not return NULL.
+      if ($access) {
```

Comment:
This should use the object notation of `$entity->access`

```php
      if (!empty($access)) {
        $accessResult = $entity->access($accessOperation ?? 'view', $accessUser, TRUE);
        $context->addCacheableDependency($accessResult);
        if ($accessResult->isDenied()) { // Not sure if this is called isDenied(). Currently not at my PC.
          return NULL;
        }
      }
```

Reply:
`$accessOperation` should already be set by the function argument default value ("view")

Reply:
Curious why returning an access result object would be better? Either way I  implemented the above, only change was the `empty($access)` I left since `$access` is a bool already.